ui/app/components/clients/line-chart.ts

```diff
@@ -81,10 +83,12 @@ export default class LineChart extends Component<Args> {
   }
   // Domains
   get yDomain() {
-    const setMax = Math.max(...this.data.map((datum) => datum.y ?? 0));
```

Comment:
See this [comment](https://github.com/hashicorp/vault/pull/24982/files#r1464093055) for more context. But I updated this for similar reasoning. 

Interfering with the domain made trends more difficult to see. Both of the following charts are rendered with the exact same data, notice how the first chart flattens the data out slightly 
## before 
<img width="371" alt="Screenshot 2024-01-23 at 3 42 40 PM" src="https://github.com/hashicorp/vault/assets/68122737/540beedd-bc33-4e06-af7c-fc13afc9db3e">

## after 
<img width="359" alt="Screenshot 2024-01-23 at 3 46 35 PM" src="https://github.com/hashicorp/vault/assets/68122737/bf588e06-0e95-4f80-b6a6-cacd61a3855c">